iotdb/src/main/java/org/apache/iotdb/db/engine/bufferwrite/BufferWriteProcessor.java

```diff
@@ -332,16 +332,16 @@ private boolean flushTask(String displayMessage, long version) {
     }
     lastFlushTime = System.nanoTime();
     // check value count
+    // waiting for the end of last flush operation.
+    try {
+      flushFuture.get();
+    } catch (InterruptedException | ExecutionException e) {
```

Comment:
What should we do? when the exception is `ExecutionException`

Reply:
I think we need to throw it.